browser/brave_tab_helpers.cc

```diff
@@ -132,6 +133,8 @@ void AttachTabHelpers(content::WebContents* web_contents) {
   }
 
   brave_wallet::BraveWalletTabHelper::CreateForWebContents(web_contents);
+
+  ntp_background_images::NTPTabHelper::CreateForWebContents(web_contents);
```

Comment:
can we create it only for NTPs if this is needed only for them?
also, do we want in OTR (i.e. incognito) contexts?

Reply:
We want to monitor tab URL changes after a user clicks on an ad and navigates away from the NTP. I don't think we require this for OTR contexts, will add a check for this.